vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridHeaderFooterRowIT.java

```diff
@@ -257,6 +259,15 @@ public void addHeaderRow_setMultiselect_disableSelection() {
                 "0", headerCells.get(0).getText());
     }
 
+    @Test
+    public void multiSelectionToSingleselect() {
+        clickButton("set-singleselect");
+        Assert.assertEquals(findElements(By.tagName("vaadin-checkbox")).stream().count(), 0);
```

Comment:
1. This is the method definition of `Assert.assertEquals`
```java
static public void assertEquals(long expected, long actual)
```
So the expected result should be the first parameter, and the actual result the second.

2. `findElements` returns a list, so you can use `size()` instead of `stream().count()`

Reply:
done

Reply:
Now that I see the fix, it occurs to me that this would be more readable:
```java
Assert.assertTrue("Error message", findElements(By.tagName("vaadin-checkbox")).isEmpty());
```
Replace "Error message" with a meaningful error message.

Reply:
fixed